websites/P/paimon.moe/presence.ts

```diff
@@ -0,0 +1,150 @@
+const presence = new Presence({
+    clientId: "859781231351693312"
+  }),
+
+  timer = Math.floor(Date.now() / 1000),
+
+  // import $ from "jquery"
+
+  characterss = [
```

Comment:
maybe name it characterList instead of characterss

Reply:
ok